task1.py

```diff
@@ -19,11 +19,12 @@
 
 if __name__ == '__main__':
     word = input("Enter the word: ")
-    if len(word) % 2 == 1:
-        a = len(word) // 2
-        let = word[a]
+    length = len(word)
+    if length % 2 == 1:
+        a = length // 2
+        result = word[a]
     else:
-        a = len(word) // 2
-        let = word[a - 1:a + 1]
+        a = length // 2
+        result = word[a - 1:a + 1]
 
-    print(let)
+    print(result)
```

Comment:
Also about 'result', this works correctly, and I don't think it's necessary to write this twice. But if it is really necessary, i'll redo it.

Reply:
> Also about 'result', this works correctly, and I don't think it's necessary to write this twice. But if it is really necessary, i'll redo it.

Hm, have no reasons not to trust you - it works if it is. But for safety reasons, and because I cannot explain why this works this way, I would declare the result variable before the if statement with an initial value of None. If it is redundant and the reviewer from Netology points on it, we will easily remove it.